float64[] x_t_cam
float64[] y_t_cam
float64[] z_t_cam

float64[] x_r_cam
float64[] y_r_cam
float64[] z_r_cam
float64[] w_r_cam
